### DISCUSSION TITLE
Is it possible for supabase to point to 3rd party Postgresql/TimescaleDB

### DISCUSSION THREAD

**Discussion by yeongren** [supabase/supabase#1891] [Questions] [answered]
Is it possible for supabase to point to 3rd party Postgresql/TimescaleDB?

I understand that the license of Timescale do not allow supabase to include in the stack. Therefore is it possible to point to a different postgresql/timescaledb?

**Comment by inian** [comment, marked as answer]
Hey @komorebi-san, the hosted solution doesn't let you bring your own database. But you can do that with the self-hosting setup. 

Timescale does have an Apache version which we can provide in the hosted setup and we have started work on testing it. Track our progress on that [here](https://github.com/supabase/postgres/pull/44)

**Reply by yeongren** [reply]
Hi @inian , do you mean for self-hosted version we can proceed to install the TimescaleDB extension as usual? And it will not break any of supabase database/auth features?

**Reply by inian** [reply]
Hey @komorebi-san, since it is just a Postgres extension, I don't think it should break anything if you are self hosting. In case it does, feel free to open an issue. 

We are starting to test on the Postgres image in the linked PR above and we don't see any problems as of now.
